azerothcore/azerothcore-wotlk
# fix(DB/creature): Adds movement for Stonewing Slayer & Phantasmal Seeker

## Changes Proposed:
- Adds a patrolling route to one Phantasmal Seeker (see TBC video)
- Adds movement to multiple Phantasmal Seeker that were stationary
- Adds movement to multiple Stonewing Slayer that were stationary

## Issues Addressed:

- Closes https://github.com/chromiecraft/chromiecraft/issues/1495
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7492

## SOURCE:

TBCC: https://youtu.be/OxwZlC6U0jU?t=204

## Tests Performed:

- Ran SQL, no errors
- checked db and in-game
- works & looks good

Patrol: 

https://user-images.githubusercontent.com/68868567/133433919-62b8ec12-5c23-4eaf-85c4-2b6266dedd3e.mp4

## How to Test the Changes:


1. go to Windrunner Village in Ghostland and check Stonewing Slayers & Phantasmal Seekers

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
